exercises/crypto-square/crypto-square.el

```diff
@@ -0,0 +1,11 @@
+;;; crypto-square.el --- Crypto Square (exercism)
+
+;;; Commentary:
+
+;;; Code:
+
+(require 'cl-lib)
```

Comment:
In the past I have tried to keep us from auto-including libraries in the solution template. I would like to not prime the student with a library inclusion and let them decide whether or not they would like to use any library code in their solution themselves.